File: app/src/main/java/cz/johrusk/showsmscode/activity/Settings_activity.java
package cz.johrusk.showsmscode.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.MenuItem;
import android.widget.FrameLayout;


import cz.johrusk.showsmscode.R;
import cz.johrusk.showsmscode.fragment.Settings_fragment;


public class Settings_activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.settings_activity);
        Toolbar toolb = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolb);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setDisplayShowHomeEnabled(true);
        getSupportActionBar().setHomeButtonEnabled(true);

        FrameLayout frame = (FrameLayout) findViewById(R.id.frame_content);
        // Display the fragment as the main content.
        getFragmentManager().beginTransaction()
                .replace(R.id.frame_content, new Settings_fragment())
                .commit();
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // handle arrow click here
        if (item.getItemId() == android.R.id.home) {
            finish(); // close this activity and return to preview activity (if there is any)
        }

        return super.onOptionsItemSelected(item);
    }


}

File: app/src/main/java/cz/johrusk/showsmscode/activity/SettingsActivity.kt
package cz.johrusk.showsmscode.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem
import cz.johrusk.showsmscode.R
import cz.johrusk.showsmscode.fragment.SettingsFragment
import kotlinx.android.synthetic.main.settings_activity.*


/**
 * Settings activity

 * @author Josef Hruska (pepa.hruska@gmail.com)
 */

class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.settings_activity)
//        ActionBar
        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.setDisplayShowHomeEnabled(true)
        supportActionBar!!.setHomeButtonEnabled(true)

        // Display the fragment as the main content.
        fragmentManager.beginTransaction().replace(R.id.frame_content, SettingsFragment()).commit()
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        // handle arrow click here
        if (item.itemId == android.R.id.home) {
            finish() // close this activity and return to preview activity (if there is any)
        }
        return super.onOptionsItemSelected(item)
    }


}

